### ISSUE TITLE
Virsh shutdown bug

### ISSUE THREAD

**Issue by diogogmt** [diogogmt/webvirt#31]
Looks like the vm instances need to have the acpid package installed:
http://askubuntu.com/questions/2825/how-do-i-use-virsh-shutdown-on-kvm-domains-to-shut-down-ubuntu-guests

**Comment by sedge** [comment]
Hmm.  This is an issue of the platform itself.  Perhaps we can display an error message suggesting this possibility?

**Comment by diogogmt** [comment]
Yes, we can display a msg just to let the user know that if the vm doesn't have the acpid package installed the shutdown command is not going to work.

**Comment by sedge** [comment]
Do you know if VIRSH supplies a specific error in this case?  If so we can just display it straight.
